articles/go-sdk.html

```diff
@@ -30,35 +30,33 @@ <h3>Install Go SDK</h3>
 </p>
 
 <p><i>
-(Before Go SDK 1.9, a <code>GOROOT</code> environment variable must be configured
+(Before Go SDK 1.9, a <code>GOROOT</code> environment variable had to be configured
 as the root path of the SDK installation.
-<a href="https://golang.org/doc/go1.9#goroot">Since Go SDK 1.9</a>,
-this environment variable becomes optional and is recommended not to set it.)
+<a href="https://golang.org/doc/go1.9#goroot">With Go SDK 1.9</a>,
+this environment variable became optional, and it is recommended to leave it unset.)
 </i></p>
 
 <p>
 Before Go SDK 1.8, a <code>GOPATH</code> environment variable must be set.
 <a href="https://golang.org/doc/go1.8#gopath">Since Go SDK 1.8</a>,
 the default value of this environment variable is the path of the <code>go</code>
 folder under the home directory of the current user.
-If you don't like the default <code>GOPATH</code>,
-you still can specify it manually.
-The <code>GOPATH</code> environment variable can contain multiple paths.
+You may specify <code>GOPATH</code> manually in order work with one (ore more) different location(s),
+an the <code>GOPATH</code> environment variable may list multiple paths.

```

Comment:
here, 
1. "in order" should be "in order to"?
2. "(ore more)" should be "(or more)"?
3. "an the" should be "and the"?